src/main/java/at/hannibal2/skyhanni/config/features/garden/EliteFarmingWeightConfig.java

```diff
@@ -29,6 +29,27 @@ public class EliteFarmingWeightConfig {
     @ConfigEditorBoolean
     public boolean leaderboard = true;
 
+    @Expose
+    @ConfigOption(name = "Leaderboard Type", desc = "Select normal or monthly weight leaderboard!")
+    @ConfigEditorDropdown
+    public Property<EliteFarmingWeightLbType> eliteLbType = Property.of(EliteFarmingWeightLbType.DEFAULT);
+
+    public enum EliteFarmingWeightLbType {
+        DEFAULT("Normal"),
```

Comment:
please have a more descriptive name for default. is this total lb? like, "all time" stats?